pkg/util/tls/tls.go

```diff
@@ -0,0 +1,87 @@
+package tls
```

Comment:
You should replace imports of `"github.com/grafana/dskit/crypto/tls"` with this one.

Reply:
@pracucci dskit/crypto/tls is actually only imported within dskit itself though, within the dskit/kv package.